configure.ac

```diff
@@ -62,8 +62,8 @@ GAP_PIC_FLAGS
 
 
 AC_LANG([C++])
-AX_CHECK_COMPILE_FLAG([-std=c++11], [
```

Comment:
we want to migrate from c++11 to c++17 and use this template with autotools to generate the configure script